pkg/kubelet/server/auth.go

```diff
@@ -72,6 +73,7 @@ func isSubpath(subpath, path string) bool {
 //	/runningPods/*	=> verb=<api verb from request>, resource=nodes, name=<node name>, subresource(s)=pods,proxy
 //	/healthz/* 		=> verb=<api verb from request>, resource=nodes, name=<node name>, subresource(s)=healthz,proxy
 //	/configz 		=> verb=<api verb from request>, resource=nodes, name=<node name>, subresource(s)=configz,proxy
+//	/statusz 		=> verb=<api verb from request>, resource=nodes, name=<node name>, subresource(s)=statusz,proxy
```

Comment:
If you make my suggestion below about moving this out of fine-grained auth, remember to remove `proxy` here too.

Reply:
done

Reply:
/lgtm